recipe/meta.yaml

```diff
@@ -1,39 +1,83 @@
 {% set name = "evidently" %}
-{% set version = "0.3.3" %}
+{% set version = "0.4.39" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/evidentlyai/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 49f3f5a724e65369f33243732dacd48aad68e3aa0bd66e3a8e7784ed5f675b5f
+  sha256: c07a65a5c07138708328dbdf1e2b369f725039f714efb393b5631db7b57fa393
+  patches:
+    - 0001-patch-test-to-exclude-experimental-features.patch
 
 build:
   number: 0
   # umap-learn isn't available on s390x
   skip: True  # [py<37 or s390x]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # missing dynaconf for py312
+  skip: True  # [py>311]
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+    # removing experimental features that are dependent on missing chromadb
+    - rm -rf $SP_DIR/evidently/experimental
 
 requirements:
+  build:
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
     - pip
     - python
     - setuptools
     - wheel
   run:
-    - nltk >=3.6.7
-    - numpy >=1.19.5
-    - pandas >=1.3.5
-    - plotly >=5.5.0
-    - pydantic >=1.9,<2
     - python
-    - pyyaml >=5.1
-    - requests >=2.19.0
-    - scikit-learn >=0.24.0
-    - scipy >=1.5.4
+    - plotly >=5.10.0
     - statsmodels >=0.12.2
-    - umap-learn >=0.5.3
+    - scikit-learn >=1.0.1
+    - pandas >=1.3.5
```

Comment:
Should be `pandas[parquet]>=1.3.5`. That means we need `pyarrow >=10.0.1` too, see https://github.com/pandas-dev/pandas/blob/v2.2.2/pyproject.toml#L71C12-L71C27

Reply:
It should be in run.

Reply:
done.